contracts/contracts/token/WOETH.sol

```diff
@@ -6,6 +6,7 @@ import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
```

Comment:
In `transferToken()` the line:


```
        require(asset_ != address(asset()), "Cannot collect OETH");
```

Should probably be "core asset" or just "asset" instead of "OETH", since this code will be used for many assets.

Reply:
good point, fixed here: https://github.com/OriginProtocol/origin-dollar/pull/2447/commits/9da27214906e5ca46f2b41b18708607411013db5